test/CollectionTest_Cpp.cc

```diff
@@ -160,8 +150,7 @@ TEST_CASE_METHOD(CollectionTest_Cpp, "C++ Default Scope", "[Collection]") {
     
     // Delete Default Collection:
     db.deleteCollection(kCBLDefaultCollectionName);
-    Collection col = db.getDefaultCollection();
-    CHECK(!col);
+    CHECK(!defaultCollection);
```

Comment:
This is not correct because defaultCollection will still hold the original pointer. The original code is correct.